GDJS/Runtime/runtimegame.ts

```diff
@@ -243,6 +251,10 @@ namespace gdjs {
       this._debuggerClient = gdjs.DebuggerClient
         ? new gdjs.DebuggerClient(this)
         : null;
+      this._captureManager = new gdjs.CaptureManager(
+        this._renderer,
+        this._options.captureOptions || {}
+      );
```

Comment:
(cf other comments, this can be done like the DebuggerClient:)

```suggestion
      this._captureManager = gdjs.CaptureManager ? new gdjs.CaptureManager(
        this._renderer,
        this._options.captureOptions || {}
      ) : null;
```